document/core/valid/instructions.rst

```diff
@@ -1633,6 +1635,22 @@ Constant Expressions
      C \vdashinstrconst \REFFUNC~x \const
    }
 
+.. math::
+   \frac{
+   }{
+     C \vdashinstrconst t.\ADD \const
+   }
+   \qquad
+   \frac{
+   }{
+     C \vdashinstrconst t.\SUB \const
+   }
+   \qquad
+   \frac{
+   }{
+     C \vdashinstrconst t.\MUL \const
+   }
```

Comment:
I'd do it like in the grammar for numeric instructions (Sec 2.4.1), as inn. Also, I would probably combine these into a single rule of the form:
```suggestion
   \frac{
     \ibinop \in \{\ADD, \SUB, \MUL\}
   }{
     C \vdashinstrconst \K{i}\X{nn}\K{.}\ibinop \const
   }
```

Reply:
Done